src/HashingPassword.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "../external/picosha2.h"
+
+class HashingPassword
+{
+    std::string hashedPassword_;
```

Comment:
hashedPassword_ will be empty before we call hashPassword function. This is not a good thing. Do we need this field or hashPassword can just return a hash?